tests/fsharp/Compiler/Language/SpanTests.fs

```diff
@@ -70,51 +70,6 @@ test ()
         
         CompilerAssert.RunScript script []
 
```

Comment:
Why are these deleted?

Reply:
These were testing a regression, which got fixed with net6.

On a second thought I probably should just change the assert.